src/app/app.component.ts

```diff
@@ -341,12 +341,16 @@ export class AppComponent implements OnInit {
               this.router.navigate(["vault"]);
             }
             break;
+          }
           case "systemSuspended":
             await this.checkForSystemTimeout(systemTimeoutOptions.onSuspend);
+            break;
```

Comment:
@addisonbeck I'm assuming these were accidentally omitted?

Reply:
This was not intentional, we definitely want break statements here.